src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPage.epics.js

```diff
@@ -3,7 +3,7 @@ import { ofType } from 'redux-observable';
 import { push } from 'connected-react-router';
```

Comment:
Made some changes there to ensure the page status is not set to "default" before the url is updated.